added cost function to minimize comddot

Ipuch opened the pull request:
I added a cost function, i think this commit should be added to your current pull request, since it concern the same type of feature.

Ipuch committed:
added cost function to minimize comddot

EveCharbie reviewed (changes requested):
I feel like this will crash if qddot is not defined as a state or a control.
I think we should add either:
if "qddot" not in nlp.states.keys() and "qddot" not in nlp.controls.keys():
	q = DynamicsFunctions.get(nlp.states["q"], states)
        qdot = DynamicsFunctions.get(nlp.states["qdot"], states)
        qddot = nlp.model.dynamics...(q, qdot, tau)
OR
        raise NotImplementedError("Need qddot to be defined as a state or a control")

Ipuch committed:
raise an Not implemented error for minimize comddot as requested by the reviewer

Ipuch commented:
done
